api/scheme.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	snapshotv1 "github.com/kubernetes-csi/external-snapshotter/client/v6/apis/volumesnapshot/v1"
 	configv1 "github.com/openshift/api/config/v1"
+	imagev1 "github.com/openshift/api/image/v1"
```

Comment:
why this change?

Reply:
There was an issue on data recolection after a failure and discussing with @csrwng we agreed in add it to the scheme.

Reply:
@jparrill that was added by this PR (https://github.com/openshift/hypershift/pull/2054), so you don't need to carry it here. Actually if you rebase your branch on main, it should go away